src/_data/documentation_categories.yml

```diff
@@ -9,6 +9,9 @@
 - title: Enriching Error Data
   slug: enriching-error-data
 
+- title: Application Performance Monitoring
+  slug: apm
```

Comment:
@MimiDumpling I went ahead and changed this to `apm` from `application-performance-monitoring`. This fixes the navigation.

Reply:
But apm isn't what the slug would normally be for this title. Perhaps it is the navigation that should change.

Reply:
@markstory what would be the appropriate slug?

Reply:
If we wanted to keep the `application-performance-monitoring` slug, I think the folder would need to be renamed to `application-performance-monitoring`. I'm not familiar with Ruby to understand how this file is loaded.

Reply:
Oh! I see. Okay. I think we should keep the apm directory name and the apm slug. It's more concise. But, maybe @markstory has a different suggestion?

Reply:
The longer name let's you get more keywords in the URLs but I don't know how much that matters anymore. The apm slug is fine by me.